src/frontend/sharedComponents/BottomSheetModal/index.tsx

```diff
@@ -15,6 +16,16 @@ export const MODAL_NAVIGATION_OPTIONS = {
   animationEnabled: false,
 };
 
+export const useSetModalNavOptions = () => {
+  const { setOptions } = useNavigationFromRoot();
+  return React.useLayoutEffect(() => {
+    setOptions({
+      presentation: "transparentModal",
+      animation: "none",
+    });
+  }, []);
+};
+
```

Comment:
unless this will change at some point, i don't think this is necessary. can instead set these options on the screens themselves e.g. when you use `WhateverStack.Screen`

similar comment about `useSetHeader` in general. that hook really only needs to be called if the component is setting something dynamic like a translated string. otherwise, it should be passed as an option when creating the screen in the stack component

Reply:
i suggest updating `MODAL_NAVIGATION_OPTIONS` in this file to the options here and then using that export in the onboarding stack as the options you pass for the relevant screens (JoinRequest and ProjectInvite modals)

Reply:
updated!